### DISCUSSION TITLE
Welcome to SimplCommerce Discussions!

### DISCUSSION THREAD

**Discussion by thiennn** [simplcommerce/SimplCommerce#1004] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ingmgumbs** [comment]
Hi, 
A few weeks ago I got to see de documentation page. But, since last week there is a 404 error:

![image](https://user-images.githubusercontent.com/22047290/207577690-428769b7-c35e-4d27-a0a8-4d504f595f8b.png)

Are you aware of it?
